components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -137,29 +123,38 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
         setTimeout(() => testSiteURL(onSuccess, onError, location.origin), 1000);
     }, []);
 
-    const dataKey = Object.keys(data);
+    const prepareDataForAccordion = () => {
+        const accData: any = [];
+        const dataKeys = Object.keys(data);
+
+        dataKeys.forEach((key: string) => {
+            const items = data[key].items.map((item) => (
```

Comment:
nice catch: that way we can add items independent from this specific use-case! 👍🏼